### PR TITLE
Add SUT: grubba

### PR DESCRIPTION
**Pull request by rmartin013**
Adding Grubba device:
https://certification.canonical.com/hardware/202308-31932/
https://barometz.admin.canonical.com/equipment/grubba

### CODE REVIEW

File: sut/grubba.conf [hunk 1]
```diff
@@ -0,0 +1,23 @@
+agent_id: grubba
+allocate_command: PYTHONIOENCODING=utf-8 PYTHONUNBUFFERED=1 snappy-device-agent noprovision
```
Comment (rmartin013) [inline sut/grubba.conf:2]: This device still requires manual provisioning (hoping to switch to MAAS next cycle), that's why I have specified noprovision mode

File: sut/grubba_snappy.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+agent_name: grubba
+device_ip: 10.229.48.48
+env:
+    HEXR_DEVICE_SECURE_ID: J3B4uh2fEFhRZGLJUE9Gtm
+    DEVICE_IP: 10.229.48.48
+maas_user: testflinger_a
+node_id: d0eff0
```
Comment (rmartin013) [inline sut/grubba_snappy.yaml:7]: I don't know the rule for how to fill the node_id, so I just took the last part of the sha256sum of the sut/grubba_snappy.yaml file... but I confirmed this is not used in other SUTs
Anyway... I am not sure those (maas_user, node_id) fields are necessary without using MAAS provisioning, maybe I should remove them as well

Comment (hum4n0id) [reply sut/grubba_snappy.yaml:7]: Correct, 'node_id' is used for MAAS provisioning.

Lgtm beyond that.

Comment (rmartin013) [reply sut/grubba_snappy.yaml]: Thanks! I removed the unnecessary stanzas

Comment (hum4n0id) [reply sut/grubba_snappy.yaml]: Merged - I will deploy the updates to the environment later today.

Comment (rmartin013) [reply sut/grubba_snappy.yaml]: Thanks!

**Pull request merged**